packages/react-i18n/src/i18n.ts

```diff
@@ -482,21 +490,36 @@ export class I18n {
   private getShortCurrencySymbol(currencyCode = this.defaultCurrency || '') {
     const currency = currencyCode || this.defaultCurrency || '';
     const regionCode = currency.substring(0, 2);
-    const info = this.getCurrencySymbol(currency);
+    let info: {symbol: string; prefixed: boolean};
+
+    // falls back to the default: symbol, for old browsers that don't support narrowSymbol
+    try {
+      info = this.getCurrencySymbol(currency, {
+        currencyDisplay: 'narrowSymbol',
+      });
+    } catch {
+      info = this.getCurrencySymbol(currency);
+    }
+
     const shortSymbol = info.symbol.replace(regionCode, '');
     const alphabeticCharacters = /[A-Za-zÀ-ÖØ-öø-ÿĀ-ɏḂ-ỳ]/;
 
+    if (currency in CurrencyShortFormException) {
+      return {
+        symbol: CurrencyShortFormException[currency],
+        prefixed: info.prefixed,
```

Comment:
I have no idea. I don't know what this value represents off the top of my head and I don't know if it changes per locale (a counter example would be fantastic showcasing prefix changing per locale). If it does change per locale then yeah keeping this as-is sounds better.

Reply:
From my investigation, this changes depending on the `locale`. And this prefixed value dictates whether the symbol would `prefix` or `suffix` the amount value (see [here](https://github.com/Shopify/quilt/blob/15d9efb38274d06f6953944b1541be9ed44b5675/packages/react-i18n/src/i18n.ts#L466)).

Looking at the `getCurrencySymbol` [here](https://github.com/Shopify/quilt/blob/15d9efb38274d06f6953944b1541be9ed44b5675/packages/react-i18n/src/utilities/money.ts#L3), and then the currencyPrefix [here](https://github.com/Shopify/quilt/blob/15d9efb38274d06f6953944b1541be9ed44b5675/packages/react-i18n/src/utilities/money.ts#L23), this is where the prefixed value ultimately comes from and it seems reliant to the locale.

So I think we could leave it as is, otherwise, for each currency, we'll need to hardcode its `prefixed` value for each `locale`. Thanks!